omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -65,45 +130,174 @@ configuration redirect the output of the following command into a file:
 To configure an HTTPS server follow
 `the nginx documentation <http://nginx.org/en/docs/http/configuring_https_servers.html>`_.
 
-Start the Gunicorn worker processes running one thread listening on 127.0.0.1:4080:
+Running OMERO.web (Unix/Linux)
+------------------------------
+
+Start the Gunicorn worker processes listening by default on 127.0.0.1:4080:
 
 ::
 
     $ bin/omero web start
-    ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
+    ... static files copied to '/home/omero/OMERO.server/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
-Additional settings can be configured from command line arguments:
+The Gunicorn workers are managed **separately** from other OMERO.server
+processes. You can check their status or stop them using the
+following commands:
 
-.. program:: omero web start
+::
+
+    $ bin/omero web status
+    OMERO.web status... [RUNNING] (PID 59217)
+    $ bin/omero web stop
+    Stopping OMERO.web... [OK]
+    Django WSGI workers (PID 59217) killed.
 
-.. option:: --workers WORKERS
+.. _download_limitation:
 
-    The number of worker processes for handling requests.
+Download limitations
+^^^^^^^^^^^^^^^^^^^^
 
-.. option:: --worker-connections WORKER_CONNECTIONS
+In order to offer users the ability to download data from OMERO.web you have
+to deploy using :ref:`gunicorn_async_configuration`.
+OMERO.web is able to handle multiple clients on a single worker
+thread switching context as necessary while streaming binary data from
+OMERO.server. Depending on the traffic and scale of the repository you should
+configure connections and speed limits on your server to avoid blocking
+resources. We recommend you run benchmark and performance tests.
+It is also possible to apply :ref:`download_restrictions` and
+offer alternative access to binary data.
 
```

Comment:
Is this advice correct?  I mean, should we not be ensuring that it works under all conditions, rather than putting the a configuration and testing burden on the admin, where it may still break anyway?